docs/content/tutorials/hello-monkey.md

```diff
@@ -0,0 +1,138 @@
+
+
+#### Tell the Monkey which exploiters to use
+Now we're going to configure the Monkey to use an exploiter. In order to do that, select **Configuration** in the navigation sidebar. Select the **Propagation** tab, and the **Exploiters** subtab. You should see a list of _Enabled exploiters_, which you'll notice, is empty. This is because we haven't installed any exploiters yet. Thankfully, the _Enabled exploiters_ list provides a link to a page where we can download exploiters, so let's follow that link and install an exploiter.
+
+![Empty exploiters list](../../images/tutorials/hello-monkey/10-empty-exploiter-list.jpg)
+
+{{% notice note %}}
+Infection Monkey does not come with exploiters pre-installed. However, it provides an easy way to download and install them.
+{{% /notice %}}
+
+You should now be at the **Plugins** page. Under the _Available Plugins_ tab you'll see a list of all the plugins that can be installed. Infection Monkey has several types of plugins. Since we're interested in installing an exploiter, let's filter this list to only show us exploiters. Select the _Type_ dropdown, and choose **Exploiter**. You should now see that the _Type_ column only has Exploiter.
+
+For this tutorial we're going to install the _SSH Exploiter_. Type "ssh" into the search field, and you should see an exploiter named **SSH**. Click the download button to install that exploiter. The button will change into a loading indicator to show that it's installing, and then to a checkmark when the plugin is installed. You can go to the _Installed Plugins_ tab and see that a plugin named **SSH** of type **Exploiter** is installed.
+
+![Filtered plugin list](../../images/tutorials/hello-monkey/11-filtered-plugin-list.jpg)
+
+Great! We've installed the SSH Exploiter, but we still need to tell the Monkey to use it. Navigate back to the **Configuration** page, and notice that the _Enabled exploiters_ list now shows _SSH Exploiter_ as an option. Check the box next to _SSH Exploiter_, leave the default option values unchanged, and then **Submit** the configuration.
+
+![Enable the SSH Exploiter](../../images/tutorials/hello-monkey/12-exploiter-enabled.jpg)
+
+Now that we've told Infection Monkey which exploiter to use, let's try running again. Go navigate to **1. Run Monkey** and select _From Island_ again. Then, let's have another look at the _Infection Map_:
+
+![Network map](../../images/tutorials/hello-monkey/8-map-scanned.jpg)
+
+Hmm. No change. It _still_ didn't attempt to exploit.
+
+This is because the SSH Exploiter requires credentials in order to run, and we haven't provided any. Let's do that now.
+
+
+#### Tell the Monkey which credentials to use
+Navigate once again to the **Configuration** page. Select the **Propagation** tab, then the **Credentials** subtab.
+
+Enter `user` into the **Identity** field, and `j688yq/pB{5=` in the **Password** field, and hit the _Save_ button.
```

Comment:
Is there any reason not to use a simpler, more human-readable password?

Reply:
I suppose not